### ISSUE TITLE
cannot find module './firebase-interfaces'

### ISSUE THREAD

**Issue by juand89** [cartant/firebase-nightlight#50]
I only have this issue when I run npm run dev on my nuxt project. The tests are not showing this error. Looking inside the node_modules there is not this file firebase-interfaces. I currently using firebase 6 version and version 2 of nuxt.

``` 
Cannot find module './firebase-interfaces'.
  > 1 | import * as firebase from "./firebase-interfaces"
````

**Comment by cartant** [comment]
I've not used or update this project for some time:

https://github.com/cartant/firebase-nightlight/blob/ceb4faee8129670c639759dedfc91c93660d424b/package.json#L52-L55

It seems that changes will be required for it to work with version 6.

**Comment by cartant** [comment]
FYI, I've updated all of the dependencies except for the Firebase ones. Those will likely take a little more time. If you are looking into this yourself, you might want to update your fork.

**Comment by cartant** [comment]
Actually, this isn't a bug. The module you refer to is an internal file within this package:

https://github.com/cartant/firebase-nightlight/blob/master/source/firebase-interfaces.d.ts

To me, it sounds like you have a problem with your 'nuxt project' and, for whatever reason, it's unable to consume an imported `.d.ts` file. Perhaps adding an empty `.js` file - with the same base name and in the same directory - would solve the problem. Try it. If it does, I can add the file before I publish an version with updated peer ranges.

**Comment by juand89** [comment]
yes indeed is a problem with my project I added an empty file with the name of firebase-interfaces and there is no more error. Thank you

**Comment by cartant** [comment]
Awesome. I'll publish a new version soon.

**Comment by cartant** [comment]
Published 3.1.1 with the updated dependencies. peer ranges and added empty file.
